README.md

```diff
@@ -183,6 +183,13 @@ end
 <% end %>
 ```
 
+### Edit and New Views with Simple Form
+```erb
+<%= simple_form_for @user, :url => users_path, :html => { :multipart => true } do |form| %>
+   <%= form.input :avatar, as: :file %>
```

Comment:
Do we actually need the `multipart`? That should be handled for us if we had a `file` input.